src/styles/components/_calendar.scss

```diff
@@ -0,0 +1,63 @@
+@function getCalendarWidth($row-Count, $cell-Size, $gap-Count, $gap-Param) {
+  @return $row-Count * $cell-Size + $gap-Count * $gap-Param;
+}
+
+@function getStartDayMargin($cell-Size, $gap-Param, $day-Num) {
+  @return $day-Num * $cell-Size + $day-Num * $gap-Param;
+}
+
+.calendar {
+  display: flex;
+  flex: 0 0 auto;
+  flex-wrap: wrap;
+  width: getCalendarWidth(7, $cellSize, 6, $gapParam);
+  padding: 10px;
+  gap: $gapParam;
+
+  &__day {
+    @include squareSidesParams($cellSize);
+    margin: 0;
+    padding: 0;
+    border: $borderParams;
+    box-sizing: border-box;
+    position: relative;
+    background-color: $mainBgColor;
+    list-style: none;
+    transition: all 0.2s ease-in;
```

Comment:
It's better to specify to what properties you apply the transition.
![image](https://github.com/mate-academy/layout_calendar/assets/63243136/ca7bc463-bdfb-4fd1-815c-2e1048ded26e)